Missing include path for MagickCore/magick-baseconfig.h

synfig 1.3.15 Linux Debian unstable amd64

MagickCore/magick-baseconfig.h isn't stored in /usr/include but in 
/usr/include/x86_64-linux-gnu/ImageMagick-7
or for imagemagick 6
/usr/include/x86_64-linux-gnu/ImageMagick-6

Maybe the best is to use pkgconfig to set CFLAGS accordingly ?

Here in MagickCore-7.Q16HDRI.pc

````
Cflags: -I${includearchdir} -I${includedir} -fopenmp -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16

````
Build error message 
````
[100%] Building CXX object src/modules/mod_magickpp/CMakeFiles/mod_magickpp.dir/main.cpp.o
cd /src/synfig-1.3.15/obj-x86_64-linux-gnu/src/modules/mod_magickpp && /usr/lib/ccache/c++  -DHAVE_CONFIG_H -DIMAGEMAGICK_CONFIG -Dmod_magickpp_EXPORTS -I/src/synfig-1.3.15/obj-x86_64-linux-gnu/src/modules/mod_magickpp -I/src/synfig-1.3.15/obj-x86_64-linux-gnu/src -I/src/synfig-1.3.15/synfig-core/src -isystem /usr/include/sigc++-2.0 -isystem /usr/lib/x86_64-linux-gnu/sigc++-2.0/include -isystem /usr/include/cairo -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/pixman-1 -isystem /usr/include/uuid -isystem /usr/include/freetype2 -isystem /usr/include/libpng16 -isystem /usr/include/glibmm-2.4 -isystem /usr/lib/x86_64-linux-gnu/glibmm-2.4/include -isystem /usr/include/giomm-2.4 -isystem /usr/lib/x86_64-linux-gnu/giomm-2.4/include -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/ImageMagick-7 -isystem /usr/include/libxml++-2.6 -isystem /usr/lib/x86_64-linux-gnu/libxml++-2.6/include -isystem /usr/include/libxml2 -isystem /usr/include/pango-1.0 -isystem /usr/include/fribidi -isystem /usr/include/harfbuzz -isystem /usr/include/mlt++ -isystem /usr/include/mlt  -g -O2 -fdebug-prefix-map=/src/synfig-1.3.15=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC   -std=gnu++11 -o CMakeFiles/mod_magickpp.dir/main.cpp.o -c /src/synfig-1.3.15/synfig-core/src/modules/mod_magickpp/main.cpp
In file included from /usr/include/ImageMagick-7/Magick++/Include.h:14,
                 from /usr/include/ImageMagick-7/Magick++.h:10,
                 from /src/synfig-1.3.15/synfig-core/src/modules/mod_magickpp/trgt_magickpp.h:38,
                 from /src/synfig-1.3.15/synfig-core/src/modules/mod_magickpp/main.cpp:40:
/usr/include/ImageMagick-7/MagickCore/magick-config.h:25:10: fatal error: MagickCore/magick-baseconfig.h: No such file or directory
   25 | #include "MagickCore/magick-baseconfig.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
````

--- a/synfig-core/src/modules/mod_magickpp/Makefile.am
+++ b/synfig-core/src/modules/mod_magickpp/Makefile.am
@@ -7,6 +7,10 @@ AM_CPPFLAGS = \
 	-I$(top_builddir) \
 	-I$(top_srcdir)/src
 
+AM_CXXFLAGS = \
+	@CXXFLAGS@ \
+	@MAGICKPP_CFLAGS@
+
 
 moduledir = @MODULE_DIR@
 
@@ -26,8 +30,8 @@ libmod_magickpp_la_LDFLAGS = \
 	-avoid-version
 
 libmod_magickpp_la_CXXFLAGS = \
-	@SYNFIG_CFLAGS@ \
-	@MAGICKPP_CFLAGS@
+	@MAGICKPP_CFLAGS@ \
+	@SYNFIG_CFLAGS@
 
 libmod_magickpp_la_LIBADD = \
 	../../synfig/libsynfig.la \

